.fides/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,312 @@
+            FROM privacyexperienceconfig
+            WHERE privacyexperienceconfig.id = :original_id
+            """
+        )
+
+        bind.execute(
+            clone_experience_config_query,
+            {
+                "id": row["experience_config_id_after"],
+                "original_id": row["experience_config_id"],
+                "version": 1,
+            },
+        )
+
+        create_experience_config_history = text(
+            """
+             INSERT INTO privacyexperienceconfighistory (id, experience_config_id, disabled, version, component, is_default, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title)
+             SELECT :id, :experience_config_id, disabled, :version, component, is_default, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title
+             FROM privacyexperienceconfig
+             WHERE privacyexperienceconfig.id = :original_id
+            """
+        )
+
+        bind.execute(
+            create_experience_config_history,
+            {
+                "id": generate_record_id("pri"),
+                "experience_config_id": row["experience_config_id_after"],
+                "original_id": row["experience_config_id"],
+                "version": 1,
+            },
+        )
+
+    # Create any needed Experience Configs that don't have coverage when we move notices and displayed_in_*
+    experience_config_df[experience_config_df["new_experience_config_needed"]].apply(
+        create_new_experience_config, axis=1
+    )
+
+    def link_new_config_to_applicable_experiences(row):
+        point_experience_to_new_config = text(
+            """
+            UPDATE privacyexperience
+            SET experience_config_id = :new_config_id
+            WHERE component = :component and region in :regions
+            """
+        )
+        bind.execute(
+            point_experience_to_new_config,
+            {
+                "component": row["component"],
+                "new_config_id": row["experience_config_id_after"],
+                "regions": tuple(row["region"]),
+            },
+        )
+
+    # Add a FK linking applicable Experiences to Experience Configs.
+    # This is the equivalent of setting "locations" on Experience Configs
+    experience_config_df[experience_config_df["new_experience_config_needed"]].apply(
+        link_new_config_to_applicable_experiences, axis=1
+    )
+
+    # Translate "disabled" concept from the Experience Config -> Privacy Experience.
+    set_experience_disabled = text(
+        """
+        UPDATE privacyexperience
+        SET disabled = privacyexperienceconfig.disabled
+        FROM privacyexperienceconfig
+        WHERE privacyexperience.experience_config_id = privacyexperienceconfig.id;
+        """
+    )
```

Comment:
Requirements have changed here.  "Disabled" is staying at the shared ExperienceConfig level instead of moving to the region-specific Experience level.